### DISCUSSION TITLE
Can the dark mode toggle be moved to settings to keep it persistent?

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1042] [Feature Requests]
Currently the dark mode switch on the website is just a toggle. Can this be moved to the user account settings so that it is persistent across different devices and browsers that clear cookies/cache on close? Currently it needs to be toggled on every time.

**Comment by mcexit** [comment]
Additionally, it would be great if it used the meta color-scheme tag[^1].

[^1]: https://web.dev/color-scheme/
